fili-core/src/main/java/com/yahoo/bard/webservice/druid/client/impl/AsyncDruidWebServiceImpl.java

```diff
@@ -356,4 +326,105 @@ public Integer getTimeout() {
     public DruidServiceConfig getServiceConfig() {
         return serviceConfig;
     }
+
+    protected Meter getHttpErrorMeter() {
+        return httpErrorMeter;
+    }
+
+    protected Meter getExceptionMeter() {
+        return exceptionMeter;
+    }
+
+    protected DruidServiceConfig getDruidServiceConfig() {
+        return serviceConfig;
+    }
+
+    /**
+     * Log request using RequestLog.
```

Comment:
This is doing much more than just logging the request, right? We should probably call out all of those different responsibilities of this method?